apps/guardian-ui/src/setupProxy.js

```diff
@@ -0,0 +1,19 @@
+const { createProxyMiddleware } = require('http-proxy-middleware');
+
+const config_path = process.env.REACT_APP_DEV_CONFIG_PATH
+  ? process.env.REACT_APP_DEV_CONFIG_PATH
+  : 'ws://127.0.0.1';
+
+module.exports = function (app) {
+  console.log(`serving up ${config_path} for frontend`);
+  app.use(
+    '/config',
+    createProxyMiddleware({
+      target: 'http://localhost:5000/dev_config',
```

Comment:
Do we need a separate webserver to serve up these configs? This `app` object is a webserver that we already can now add behavior to ourselves, we can just do any logic of pulling in JSON files and serving them here.